harness/determined/cli/experiment.py

```diff
@@ -1294,6 +1323,26 @@ def experiment_id_arg(help: str) -> Arg:  # noqa: A002
                         Arg("max_slots", type=none_or_int, help="max slots"),
                     ],
                 ),
+                Cmd(
+                    "log-retention",
+                    set_log_retention,
+                    "set `log-retention-days` for an experiment",
+                    [
+                        experiment_id_arg("experiment ID"),
+                        Arg(
+                            "--days",
+                            type=none_or_int,
+                            help="from the time of creation, number of days to "
```

Comment:
Since we're enforcing a range in the check, add that range to the help text. 
`allowed range:  -1 to 32767.`

Reply:
fixed!